flink-autoscaler/src/test/java/org/apache/flink/autoscaler/JobVertexScalerTest.java

```diff
@@ -775,11 +931,119 @@ public void testSendingIneffectiveScalingEvents(Collection<ShipStrategy> inputSh
         assertThat(event.getReason()).isEqualTo(INEFFECTIVE_SCALING);
     }
 
+    @Test
+    public void testNumPartitionsAdjustment() {
+        final int minParallelism = 1;
+        final int maxParallelism = Integer.MAX_VALUE;
```

Comment:
They are `parallelismLowerLimit` and `parallelismUpperLimit`.

It's better to distinguish `parallelismUpperLimit` and `maxParallelism` here.